radio/src/fonts.h

```diff
@@ -36,7 +36,7 @@ void loadFontCache();
 extern const pm_uchar font_5x7[];
 extern const pm_uchar font_10x14[];
 
-#if defined(BOLD_FONT) && (!defined(CPUM64) || defined(TELEMETRY_NONE)) && !defined(BOOT)

```

Comment:
I'm not the author of this line.
I propose only to replace it by something more understandable.